ontology.ttl

```diff
@@ -458,8 +461,9 @@ owl:versionIRI rdf:type owl:AnnotationProperty .
          rdfs:subClassOf :NumericParameter ,
                          [ rdf:type owl:Restriction ;
                            owl:onProperty :value ;
-                           owl:allValuesFrom xsd:integer
-                         ] .
+                           owl:allValuesFrom xsd:integer, xsd:int
```

Comment:
as far as I undersand owl:allValuesFrom can appear only once, so this is not the proper way (at least Protege doesn't understand this) and some form of concatenation may be used

But anyway, xsd:int denotes signed 32 bit integer and xsd:integer is integer unbounded value. So one is a subset of the other and strictly speaking we can use only one